server/schema.ts

```diff
@@ -0,0 +1,84 @@
+import { makeExecutableSchema } from "graphql-tools";
+import * as fetch from "isomorphic-unfetch";
+
+// Types
+const typeDefs = `
+  type Query {
+    categories: [Category]
+    exercises(category: Int!) : [Exercise]
+    exercise(id: Int!) : Exercise
+    exerciseimage(id: Int!) : [ExerciseImage]
+  }
+
+  type Category {
+    id: Int!
+    name: String!
+  }
+
+  type Exercise {
+    id: Int!
+    license_author: String
+    status: String
+    description: String
+    name: String
+    name_original: String
+    creation_data: String
+    uuid: String
+    license: Int
+    category: Int
+    language: Int
+    muscles: [Int]
+    muscles_secondary: [Int]
+    equipment: [Int]
+    image: [ExerciseImage]
+  }
+
+  type ExerciseImage {
+    id: Int!
+    license_author: String
+    status: String
+    image: String
+    is_main: Boolean
+    license: Int
+    exercise: Int!
+  }
+`;
+
+// Resolvers
+const resolvers = {

```

Comment:
I'd define a helper method as every request seems to have to do the same thing anyway:
```typescript
const getJSON = async (url: string) => {
  const res = await fetch(url);
  return res.json();
};
```